pkg/operator/ceph/config/network_test.go

```diff
@@ -196,3 +196,15 @@ func TestGetNetworkRange(t *testing.T) {
 	networkRange = getNetworkRange(netConfig)
 	assert.Equal(t, "192.168.0.0/24", networkRange)
 }
+
+func TestGetMultusNamespace(t *testing.T) {
+	// TEST 1: When namespace is specified with the NAD
+	namespace, nad := GetMultusNamespace("multus-ns/public-nad")
+	assert.Equal(t, "multus-ns", namespace)
+	assert.Equal(t, "public-nad", nad)
+
+	// TEST 2: When only NAD is specified
+	namespace, nad = GetMultusNamespace("public-nad")
+	assert.Empty(t, namespace)
+	assert.Equal(t, "public-nad", nad)
+}
```

Comment:
How about adding a couple checks for invalid input?

Reply:
`GetMultusNamespaces()` is just taking a string and splitting it with `/` so that the first part is the namespace and the second is the name of NAD. If not split into two then the complete string is the name of the NAD.
The function doesn't check for errors so skipping the checks for invalid input.

Reply:
I see, test 2 covers the case where there is no `/`